packages/rancher-logging/overlay/templates/logging-rke1.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
 spec:
   controlNamespace: {{ .Release.Namespace }}
   fluentbit:
+    image:
+      repository: {{ .Values.images.fluentbit.repository }}
```

Comment:
Since this is in a template, we can just prepend registry helper to the repository, otherwise we have to update the banzai template to do it. And then follow that pattern everywhere else.